bemoji

```diff
@@ -156,7 +156,7 @@ _typer() {
     elif [ -n "$WAYLAND_DISPLAY" ] && command -v wtype >/dev/null 2>&1; then
         wtype -s 30 "$totype"
     elif [ -n "$DISPLAY" ] && command -v xdotool >/dev/null 2>&1; then
-        xdotool type "$totype" --delay 30
+        xdotool --delay 30 type "$totype"
```

Comment:
Tried it on an xorg server today and this results in error: `xdotool: unrecognized option '--delay'`.

As per the man-page, I think it should rather be `xdotool type --delay 30 <text>`, with the delay coming after the type option. Can you test that this is the correct command?

Reply:
You're definitely right, my fault, I've fixed it. Right now it works!